### PR TITLE
[WIP] Rackspace Migration Manager: Add landing page info for review by PM

### LINKED ISSUE #1834: [MVP] Rackspace Migration Manager: Create index.md for landing page
Something similar to https://support.rackspace.com/how-to/dedicated-hosting/

### PR DESCRIPTION
**Pull request by cyrichardson**
Fixes #1834 

FAWS tab on How-To landing page will not go live until release currently planned for 3/7

Possible text for the landing page.

Asking Trent to review before getting it to Kelly.

### CODE REVIEW

**Comment by cyrichardson** [comment]
Fixes #1835

**Comment by cyrichardson** [comment]
Fixes #1834

File: content/FAWS/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+product: Fanatical Support for AWS
+product_url: fanatical-support-for-aws
+---
+
+Many businesses are seeking assistance from certified experts in getting the most out of AWS. Migration, architecture design, security, and operations can take your focus away from your core business.
+
+Fanatical Support for AWS is the answer for businesses facing these challenges. Rackspace blends technology and automation plus human experts to deliver ongoing architecture, security, and 24x7x365 operations backed by AWS certified engineers and architects.
+
+Rackspace has created a world-class migration experience that provides the tooling, process, and support to enable you to self-service your migration
+from Rackspace Public Cloud to AWS. Rackspace also provides Rackspace Professional Services with the capability to help you with your migration.
```
Comment (trentgillaspie) [inline content/FAWS/index.md]: should have additional call out to FAWS, like "...self-service migration from Rackspace Public Cloud to AWS, while maintaining the Fanatical Support you know and love from Rackspace. Rackspace also provides..."

Comment (cyrichardson) [inline content/FAWS/index.md]: Added

**Comment by cyrichardson** [comment]
@kmsholcomb When you have time, please edit these paragraphs that will be the landing page text for the Fanatical Support for AWS tab on the How-To center. Thanks!

**Comment by cyrichardson** [comment]
Ready to merge

**Comment by kmsholcomb** [comment]
@cyrichardson. I made as much hay as I could from three paragraphs! :) 

[Fanatical Support for AWS_landing page text-editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/775537/Fanatical.Support.for.AWS_landing.page.text-editsKH.pdf)
[Fanatical Support for AWS_landing page text-editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/775536/Fanatical.Support.for.AWS_landing.page.text-editsKH.docx)

**Comment by cyrichardson** [comment]
@kmsholcomb Thank you!

**Comment by stephamon** [comment]
May release

**Comment by stephamon** [comment]
Release now mid-to-late June

**Comment by cyrichardson** [comment]
@stephamon Release is now mid to late July. 

Also have a content folder for this now call rackspace-migration-manager rather than FAWS. Will need to update and change the folder this goes into when we got the go ahead on Rackspace Migration Manager.

**Comment by cyrichardson** [comment]
Closing the PR for FAWS content folder - created new PR #2085 for rackspace-migration-manager folder.

**Pull request closed without merging**